custom-footer.html

```diff
@@ -1,3 +1,6 @@
+<div class="container">
+    <p><a href="#link-to-statement">Accessibility statement</a></p>
```

Comment:
```suggestion
    <p><a href="https://www.notifications.service.gov.uk/accessibility-statement">Accessibility statement</a></p>
```

Reply:
yeah i left the href here generic so that other GDS teams don't link to our statement